modules/engine/src/debug/copy-texture-to-image.ts

```diff
@@ -44,10 +43,7 @@ export function copyTextureToDataUrl(
   source: Texture | Framebuffer,
   options: CopyTextureToImageOptions = {}
 ): string {
-  const {
-    sourceAttachment = GL.COLOR_ATTACHMENT0, // TODO - support gl.readBuffer
```

Comment:
This default is set in WebGLDevice's implementation of `readPixelsToArrayWebGL`. Avoid depending on `@luma.gl/constants` here.